agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -341,7 +341,9 @@ void ap_manager_thread::after_select(bool timeout)
             return;
         } else {
             LOG(INFO) << "waiting to attach to " << ap_wlan_hal->get_radio_info().iface_name;
-            UTILS_SLEEP_MSEC(200);
+            // Set the sleep to little less than the select timeout so we won't get awake timeout
+            // print to the log on the socket thread.
+            UTILS_SLEEP_MSEC(SELECT_TIMEOUT_MSC * 0.9);
```

Comment:
This change has nothing to do with this PR.
Also, it's fine for the threads to work "harder" while trying to attach, so I don't think it's necessary at all.

Reply:
Yes but it is not working hard, it sleeps, and it adds the annoying warning messages to the log while attaching.

Reply:
Why does it show the warning if it's sleeping for 200ms?

Reply:
It used to show because the select timeout was 200ms too. Anyway raising it is a good idea since there will be fewer attach prints, and also I wanted to set it as a number which is less than than the select timeout to avoid the warning messages.